documentation/src/main/asciidoc/upgrading/upgrading.asciidoc

```diff
@@ -20,6 +20,13 @@ as milliseconds.
 === Event log changes
 Several new event log messages have been added, and one message has been removed (ISPN100013).
 
+=== Max Idle Expiration Changes
+The max idle expiration information is and has been replicated to the entire cluster. However when an entry expired via max idle this was not replicated (only removing it locally). Max idle has been enhanced to now be clusterwide when it occurs. This includes ensuring that max idle expiration is applied across all owners (meaning if another node has accessed the entry it will prevent a different owner from thinking the entry is expired).
```

Comment:
Reword it a bit: the first sentence is a bit confusing.

Reply:
Sure. Was trying to keep it short, but might have to be a bit more detailed.